src/libraries/System.Net.WebSockets.Client/tests/ClientWebSocketOptionsTests.cs

```diff
@@ -72,12 +72,9 @@ public async Task Proxy_ConnectThruProxy_Success(Uri server)
             string proxyServerUri = System.Net.Test.Common.Configuration.WebSockets.ProxyServerUri;
             if (string.IsNullOrEmpty(proxyServerUri))
             {
-                _output.WriteLine("Skipping test...no proxy server defined.");
```

Comment:
This should probably instead be `throw new SkipTestException("No proxy server defined.");`